sdks/java/core/src/test/java/org/apache/beam/sdk/io/TextIOTest.java

```diff
@@ -404,6 +406,37 @@ public void testCompressedRead() throws Exception {
 
   @Test
   @Category(NeedsRunner.class)
```

Comment:
(@jbonofre no action needed, this is just copying another test and has no new issues w.r.t. this comment. -- comments below are just a heads-up)

@tgroh I'm sure you're thinking about these tests... Needs runner and also needs local filesystem to be the same system as pipeline. As we improve things, should we be (e.g.) using IOChannelUtils and such instead to create the temp files?